src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -334,6 +326,38 @@ public virtual void ValidateUserInfoResponse(OpenIdConnectProtocolValidationCont
         }
 
         /// <summary>
+        /// Calls <see cref="Dispose(bool)"/> and <see cref="GC.SuppressFinalize"/>
+        /// </summary>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <summary>
+        /// Calls <see cref="HashAlgorithm.Dispose()"/> to release this managed resources.
+        /// </summary>
+        /// <param name="disposing">true, if called from Dispose(), false, if invoked inside a finalizer.</param>
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!this._disposed)
+            {
+                if (disposing)
+                {
+                    foreach (var item in _hashAlgorithmMap)
+                    {
+                        if (item.Value != null)
```

Comment:
Have a look at this: https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md. We decided to follow these guidelines long time back. In this case, no { } if there is only one line inside if.